pkg/quarkus/v1alpha/scaffolds/internal/templates/makefile.go

```diff
@@ -62,8 +62,11 @@ func (f *Makefile) SetTemplateDefaults() error {
 }
 
 const makefileTemplate = `
-# Image URL to use all building/pushing image targets
+# Image URL to use for all building/pushing image targets
 IMG ?= {{ .Image }}
+# Operating system and architecture to use for building image
+OS ?= {{ .OS }}
+ARCH ?= {{ .Arch }}
```

Comment:
One thing to note here is that with the Go templates, to use the notation `{{ .X }}` the field has to exist on the object being used to render the template. When I attempted to run `operator-sdk init ...` with the changes on this branch I received the error:
```
FATA[0000] failed to initialize project: unable to scaffold with "quarkus.javaoperatorsdk.io/v1-alpha": template: *templates.Makefile:5:9: executing "*templates.Makefile" at <.OS>: can't evaluate field OS in type *templates.Makefile 
```

To fix this I think we need to:
1. Update the `Makefile` struct definition:
https://github.com/operator-framework/java-operator-plugins/blob/0eed476d2b7548daf184fb7a836ab8ac8e0dead9/pkg/quarkus/v1alpha/scaffolds/internal/templates/makefile.go#L25-L37
to have those fields.

2. Update the `Makefile.SetTemplateDefaults()` function:
https://github.com/operator-framework/java-operator-plugins/blob/0eed476d2b7548daf184fb7a836ab8ac8e0dead9/pkg/quarkus/v1alpha/scaffolds/internal/templates/makefile.go#L39-L62
to set the default values if they are not specified. I think a good default here would be `OS := linux` && `Arch := amd64`.